content/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/managing-privately-reported-security-vulnerabilities.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Managing privately reported security vulnerabilities
-intro: Repository maintainers can manage security vulnerabilities that have been privately reported to them by security reseachers for repositories where private vulnerability reporting is enabled.
+intro: Repository maintainers can manage security vulnerabilities that have been privately reported to them by security researchers for repositories where private vulnerability reporting is enabled.
```

Comment:
#### Before
https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/managing-privately-reported-security-vulnerabilities
<img width="739" alt="image" src="https://github.com/github/docs/assets/2119212/6c198e04-1812-4ce1-82a7-27fedecab5fe">

#### After
https://docs-30890-94eb78.preview.ghdocs.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/managing-privately-reported-security-vulnerabilities
<img width="719" alt="image" src="https://github.com/github/docs/assets/2119212/381487b8-5901-4340-94ee-0b005a69f7bc">